src/main/java/com/fvv/bookstore/menu/Menu.java

```diff
@@ -422,6 +429,93 @@ private void createSubMenuEmployee() {
 		}
 	}
 	
+	/**
+	 * Method to create and show the reports sub menu.
+	 */
+	private void createSubMenuReports() {
+		while(true) {
+			String input = JOptionPane.showInputDialog(
+					null, "Reports Section" + Constants.LINE_SEPARATOR + Constants.LINE_SEPARATOR +
+					"Select an option below:" + Constants.LINE_SEPARATOR +
+					"1 - Individuals" + Constants.LINE_SEPARATOR +
+					"2 - Products - Stock to Replace" + Constants.LINE_SEPARATOR +
```

Comment:
It should be "2 - Stock - Products to Replace"

Reply:
ok